src/servers/graphql-old-server.ts

```diff
@@ -228,12 +229,24 @@ const resolvers = {
         return true
       },
     }),
-    noauthAddInvoice: async (_, { username, value }, { logger }) => {
-      const wallet = await getWalletFromUsername({ username, logger })
-      return wallet.addInvoice({ selfGenerated: false, value })
+    noauthAddInvoice: async (_, { username, value }) => {
+      const lnInvoice = await addInvoiceForRecipient({
```

Comment:
I guess this is the problem. `value` can be null/undefined

Reply:
Yup looks like it's a problem with how we're testing with use-case separation being incompatible with the old schema.

I can look into a fix for this